docs/datascience/data-science-tutorial.md

```diff
@@ -247,7 +247,7 @@ A neural network is a model that uses weights and activation functions, modeling
 1. With the model built and trained its now time to see how it performs against the test data.
 
    ```python
-   y_pred = model.predict_classes(X_test)
+   y_pred = np.argmax(model.predict(X_test), axis=-1)
    print(metrics.accuracy_score(y_test, y_pred))
```

Comment:
I receive the same results with Keras [version `2.6.0`](https://github.com/keras-team/keras/wiki/Keras-2.0-release-notes) and they highly differ from the tutorial results (~80% vs. ~61%).

Python code:
```
   y_pred = np.argmax(model.predict(X_test), axis=1)
   print(metrics.accuracy_score(y_test, y_pred))
   print(metrics.classification_report(y_test, y_pred))
```

Output:
```
0.6124401913875598
              precision    recall  f1-score   support

           0       0.61      1.00      0.76       128
           1       0.00      0.00      0.00        81

    accuracy                           0.61       209
   macro avg       0.31      0.50      0.38       209
weighted avg       0.38      0.61      0.47       209
```

Reply:
Tốt

> `

Reply:
Tốt